script.js

```diff
@@ -0,0 +1,85 @@
+// Creating featured stars section elements
+const featuredStars = document.getElementById('featured-stars');
+const featuredTitle = document.createElement('h2');
+const orangeLine = document.createElement('span');
+const starSection = document.createElement('div');
+const starDiv = [];
+const starImage = [];
+const starInformation = [];
+const starName = [];
+const starAbout = [];
+const starLine = [];
+const starDescription = [];
+// Setting main elements
+featuredTitle.innerHTML = 'Featured Stars';
+orangeLine.classList.add('orange-line');
+featuredStars.appendChild(featuredTitle);
+featuredStars.appendChild(orangeLine);
+featuredStars.appendChild(starSection);
+// Creating an array of objects to store stars data
+const starData = [
+  {
+    name: 'Ash Ketchum',
+    image: 'Assets/images/featured-stars/ash.jpg',
+    about: 'Pokemon trainer and champion',
+    description: 'Aspiring to be the best trainer of all time, Ash is always seeking to improve his skills.',
+  },
+  {
+    name: 'Professor Oak',
+    image: 'Assets/images/featured-stars/professor-oak.png',
+    about: 'Pokemon trainer, professor and researcher',
+    description: 'Oak studies the human-pokemon relationship and help new trainers start a pokemon journey.',
+  },
+  {
+    name: 'Pikachu',
+    image: 'Assets/images/featured-stars/pikachu.png',
+    about: 'Pokemon',
+    description: 'Electric-type pokemon and Pokemon biggest star.',
+  },
+  {
+    name: 'Cynthia',
+    image: 'Assets/images/featured-stars/cynthia.png',
+    about: 'Pokemon trainer, champion and archeologist',
+    description: 'One of the greatest trainers of all time, champion of the Sinnoh League.',
+  },
+  {
+    name: 'Bulbasaur',
+    image: 'Assets/images/featured-stars/bulbasaur.png',
+    about: 'Pokemon',
+    description: 'Grass/Posion-type pokemon and the best first option.',
+  },
+  {
+    name: 'Ditto',
+    image: 'Assets/images/featured-stars/ditto.png',
+    about: 'Pokemon',
+    description: 'Normal-type pokemon, can transform into other pokemons. Maybe there are only dittos out there.',
+  },
+];
+// A for loop to set star elements i times
+for (let i = 0; i < 6; i += 1) {
```

Comment:
Your code iterates through the speakers list and creates HTML elements with the data from each object 🔥 Great work!